www/src/pages/table/forms/TransformationForm.tsx

```diff
@@ -0,0 +1,289 @@
+import { WithStyles, withStyles } from "@material-ui/styles";
+import {
+  Button,
+  Radio,
+  Tag,
+  Space,
+  Table,
+  Modal,
+  Form,
+  Col,
+  Row,
+  Select,
+  InputNumber,
+} from "antd";
+import { observer } from "mobx-react";
+import { ActionType } from "@ant-design/pro-table";
+import { useRef, useState } from "react";
+import Editor from "@monaco-editor/react";
+import {
+  TransformationResult,
+  useStores,
+  Transformation,
+  Table as TableModel,
+} from "../../../models";
+const styles = {};
```

Comment:
add empty line above, -- but if we don't use styles -- remove them.

Reply:
removed